### PR TITLE
Validate BonkCola (BONKCOLA)

### PR DESCRIPTION
**Pull request by dogecolafinance**
# Validate [BONKCOLA](https://solscan.io/token/Corri6VKgvE6ZRamtuBpjVdfaaATNrbSpLVuZN1n8773)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/bonk_cola/status/1734239598786515023?s=20
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/bonkcola

## Validation (Please check off boxes):
- [X] The metadata provided in the PR matches what is on-chain (Mandatory)
- [X] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [X] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [X] My change matches the format in the file (no spaces between fields).
- [X] My token is already live and trading on Jupiter.
- [X] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by dogecolafinance** [comment]
team please review. reach out to https://t.me/dogeglass (DogeCola founder) if you have questions

File: validated-tokens.csv [hunk 1]
```diff
@@ -1038,3 +1038,4 @@ Lamas Finance,LMF,LMFzmYL6y1FX8HsEmZ6yNKNzercBmtmpg2ZoLwuUboU,9,https://www.lama
 Basilisk,BSLSK,EMAb1cexvwrMeViyYiK8941V7SGWiXsyhMREBcswggPw,2,https://i.postimg.cc/d0j6rRWH/Dark-BASLSK-666x666.png,true
 Rocky,ROCKY,4icEZCrEYNop2ZaMMCkRHaNzkt6xG9BpijMCQV7mpw6Z,9,https://i.ibb.co/X8r9531/rockylogo-transparents.png,true
 Dogelon (Wormhole),ELON,7ZCm8WBN9aLa3o47SoYctU6iLdj7wkGG5SV2hE5CgtD5,4,https://dogelonmars.com/img/logo.png,true
+BonkCola,BONKCOLA,Corri6VKgvE6ZRamtuBpjVdfaaATNrbSpLVuZN1n8773,9,https://ibb.co/C95KKzN,true
```
Comment (Timoon21) [inline validated-tokens.csv:1041]:
```suggestion
BonkCola,BONKCOLA,Corri6VKgvE6ZRamtuBpjVdfaaATNrbSpLVuZN1n8773,9,https://ipfs.io/ipfs/Qmaxh1o9hYtAMqNjQcVgsGjCsuGu9CojFTxsjRt8hX3wa4,true
```

**Review by Timoon21 — APPROVED** [review]

**Comment by dogecolafinance** [comment]
@Timoon21 I have just resolved the conflicts. Can you guys approve again?

**Comment by dogecolafinance** [comment]
@Timoon21 I have just resolved the conflicts. Can you guys approve again?

**Comment by Jymfletzfur** [comment]
Insufficient Volume

**Pull request closed without merging**